### DISCUSSION TITLE
After UBL printer in weird state

### DISCUSSION THREAD

**Discussion by patofoto** [Jyers/Marlin#1189] [General Discussion]
After I use the Bed Visualizer plugin to build a mesh of my bed, my printer display is stuck as if I was printing something:

![IMG_4481](https://user-images.githubusercontent.com/37857628/126537504-6a7eecfa-94e2-47e8-b33c-da364cc91c39.jpeg)

I try to cancel the "print" but the confirm button has no effect. 

![IMG_4482](https://user-images.githubusercontent.com/37857628/126537581-27ba352e-8d8e-4492-8d12-7a8c03446cd6.jpeg)

Wondering if it is the G-Code I am using for the mesh build:

```
G28       ; home all axes
M155 S30  ; reduce temperature reporting rate to reduce output pollution
M190 S60  ; (optional) wait for the bed to get up to temperature
G29 P1    ; automatically populate mesh with all reachable points
G29 P3    ; infer the rest of the mesh values
G29 P3    ; infer the rest of the mesh values again
@BEDLEVELVISUALIZER	; tell the plugin to watch for reported mesh
M420 S1 V ; enabled leveling and report the new mesh
M500      ; save the new mesh to EEPROM
M155 S3   ; reset temperature reporting
M300 S1000 P500 ; chirp to indicate bed mesh leveling is done and saved
```

Any help would be appreciated

**Comment by deepusudhakar** [comment]
Seeing the same issue on my end with the same GCODE (grabbed from the plugin’s FAQ). Did you ever find a solution to this problem?
